spec/views/hq/partners/index_spec.rb

```diff
@@ -1,11 +1,34 @@
 require 'rails_helper'
 
+
 RSpec.describe "hq/partners/index.html.erb", type: :view do
   context 'partners exist' do
+    let(:partners) do
+      17.times do
```

Comment:
http://mislav.uniqpath.com/rails/will_paginate-view-testing/ - the gem's author explains how to test without hitting the db at all.
That article also addresses my concern about the fact that the tests `calls will_paginate` and `calls page_entries_info` might just be too low-level. This is a view spec - should we test that it renders required data or selectors, rather than that it simply makes method calls?